README.md

```diff
@@ -1,76 +1,88 @@
 # GRASS Addons git repository
 
-## How to get write access here?
+## How to get write access here
 
 Read access is granted to the public, write access
-must be requested, see here for details: \
-   https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository
+must be requested, see here for details:
+<https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository>
 
-However, it is also possible to fork this repository, insert your AddOn or fix an existing one 
-in a new branch and finally open a [pull request](https://help.github.com/en/articles/about-pull-requests).
+However, it is also possible to fork this repository, insert your AddOn or fix
+an existing one in a new branch and finally open
+a [pull request](https://help.github.com/en/articles/about-pull-requests).
 
 In either case, please read the submitting rules at the bottom of this page.
 
-## How to get the AddOn code?
+## How to get the AddOn code
 
 Clone of the entire AddOns git repository:
 
-    git clone https://github.com/OSGeo/grass-addons.git grass_addons
+```
+git clone https://github.com/OSGeo/grass-addons.git grass_addons
+```
 
 ## How to install or remove AddOns in your GRASS installation
 
-The simplest way to install GRASS GIS AddOns is to use the `g.extension` 
-module: \
-https://grass.osgeo.org/grass78/manuals/g.extension.html
+The simplest way to install GRASS GIS AddOns is to use the `g.extension`
+module:
+<https://grass.osgeo.org/grass78/manuals/g.extension.html>
 
-##  How to compile AddOn code
+## How to compile AddOn code
 
-### C code/Scripts, with GRASS source code on your computer:
+### C code/Scripts, with GRASS source code on your computer
 
-  Preparations (assuming source code in $HOME/grass78/):
-    (if you have already build GRASS from source you don't need to do this
-    again. If adding to a binary install the versions must match exactly.
-    For a git clone this means that the main GRASS binary and source
-    code versions (GRASS GIS 6 or 7) must match.)
-    
-    ./configure # [opionally flags]
-    make libs
+Preparations (assuming source code in $HOME/grass78/):
+(if you have already build GRASS from source you don't need to do this
```

Comment:
```suggestion
(if you have already built GRASS from source you don't need to do this
```

Reply:
I have also noticed that and wanted to - for the sake of the git history purity - open another PR for that, but yes, I guess it is just an annoying overhead to split these small changes into fifteen smaller ones.